bundles/org.openhab.core.addon.marketplace/src/main/resources/OH-INF/i18n/marketplace.properties

```diff
@@ -0,0 +1,4 @@
+system.config.marketplace.apiKey.label = API Key for community.openhab.org
+system.config.marketplace.defaultSource.description = Specify the API key to use on the community forum (for staff and curators - this allows for instance to see content which is not yet reviewed or otherwise hidden from the general public). Leave blank if you don't have one.
+
+service.system.marketplace.label = Community Marketplace
```

Comment:
Should we maybe simply call it "Marketplace" here?

Reply:
My view that the "community marketplace" (whose content comes from the community forum) to be only one of the possible implementations of a "marketplace", which may eventually include other sources - you could have other non-community marketplaces, for example commercial vendors.

Reply:
Ok, fair enough.